src/main/java/com/googlecode/lanterna/gui2/AbstractBorder.java

```diff
@@ -50,23 +50,38 @@ public Border setSize(TerminalSize size) {
 
     @Override
     public LayoutData getLayoutData() {
+        if(getComponent() == null) {
+            return super.getLayoutData();
+        }
         return getComponent().getLayoutData();
     }
 
     @Override
     public Border setLayoutData(LayoutData ld) {
-        getComponent().setLayoutData(ld);
+        if(getComponent() == null) {
+            super.setLayoutData(ld);
+        } else {
+            getComponent().setLayoutData(ld);
+        }
         return this;
     }
 
     @Override
     public TerminalPosition toBasePane(TerminalPosition position) {
-        return super.toBasePane(position).withRelative(getWrappedComponentTopLeftOffset());
+        TerminalPosition terminalPosition = super.toBasePane(position);
+        if(terminalPosition == null) {
+            return null;
```

Comment:
Not sure, if null is best choice here...
A border without content might just return its argument, maybe augmented for it's own border "thickness" (typically 1 or maybe 0).

Given that it was an NPE before, anything else would be ok...